lib/index.d.ts

```diff
@@ -23,10 +29,10 @@ export class StateController<State> {
   /* A readonly version of controller's state */
   readonly state: State;
 
-  /* An initial default property bag for the controller's state implemented as get defaultState()*/
+  /* An initial default property bag for the controller's state implemented as get defaultState() */
   readonly defaultState: State;
 
-  /** Update the state by passing in a property bag */
+  /* Update the state by passing in a property bag */
```

Comment:
why? `/**` double star shows up in documentation and autocomplete intellisense, help. Single ones don't.

If you feel strongly, then feel free to make it multiline e.g.

```
/**
 * xyz
 */
```

Reply:
I was just going for consistency -- if there's a functional advantage `/**` makes sense to me